docs/examples/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Web related
+*/lib/generated_plugin_registrant.dart
+
+# Platforms
+*/android
+*/ios
+*/web
+*/macos
+*/test
```

Comment:
Doesn't it need **/*/android etc?

Reply:
I think this too, that way we cover everything, regardless the level of example folder tree

Reply:
I didn't want be that invasive and possibly ignoring proper project directories.

Reply:
We have for example the box2d and effects directory now (and there will probably be more), I think it will give more than the amount of invasive it will be.

Reply:
Do these sub examples really need to be separate projects?

So far most examples separate different ocurrences in files inside the same project, IDK why these are different.

Reply:
It will be much easier for the user to run them if we have many separate ones. I think at the rate Flame is growing now there will be loads of examples coming in for new features. Both Box2D and Effects will have more coming in within a week.

Reply:
The chances of someone naming their folders something like that are really low too, and if they did I think it should be rejected on a PR level since it will be confusing to have the same names as are part of the basic flutter structure.

Reply:
I am with @spydon on this, I don't this will be invasive, this is our examples structure, and we should keep a standard for it, otherwise it can grow out of our hands, so I would prefer to cover all the sub directories using the glob suggested by spydon.

Reply:
wilco